.gitlab-ci.yml

```diff
@@ -169,17 +171,16 @@ variables:
   # Build images versions
   # To use images from datadog-agent-buildimages dev branches, set the corresponding
   # SUFFIX variable to _test_only
-  DATADOG_AGENT_BUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_BUILDIMAGES: v44808106-d8c4f8af
-  DATADOG_AGENT_WINBUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_WINBUILDIMAGES: v44808106-d8c4f8af
-  DATADOG_AGENT_ARMBUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_ARMBUILDIMAGES: v44808106-d8c4f8af
-  DATADOG_AGENT_SYSPROBE_BUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_SYSPROBE_BUILDIMAGES: v44808106-d8c4f8af
-  DATADOG_AGENT_BTF_GEN_BUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_BTF_GEN_BUILDIMAGES: v44808106-d8c4f8af
-
+  DATADOG_AGENT_BUILDIMAGES_SUFFIX: "_test_only"
+  DATADOG_AGENT_BUILDIMAGES: v45218254-7d954270
+  DATADOG_AGENT_WINBUILDIMAGES_SUFFIX: "_test_only"
+  DATADOG_AGENT_WINBUILDIMAGES: v45218254-7d954270
+  DATADOG_AGENT_ARMBUILDIMAGES_SUFFIX: "_test_only"
+  DATADOG_AGENT_ARMBUILDIMAGES: v45218254-7d954270
+  DATADOG_AGENT_SYSPROBE_BUILDIMAGES_SUFFIX: "_test_only"
+  DATADOG_AGENT_SYSPROBE_BUILDIMAGES: v45218254-7d954270
+  DATADOG_AGENT_BTF_GEN_BUILDIMAGES_SUFFIX: "_test_only"
+  DATADOG_AGENT_BTF_GEN_BUILDIMAGES: v45218254-7d954270
```

Comment:
`test_only` are images generated by dev branches from buildimages, could you use the image generated on the main pipeline ?